### ISSUE TITLE
Video download doesn't work

### ISSUE THREAD

**Issue by test2a** [SuhasDissa/MemerizeApp#1]
It seems only images are being downloaded. 
If we click on download, it opens redditsave.com link in the browser. 

It should do so transparently like images

**Comment by SuhasDissa** [comment]
Downloading reddit videos need some additional libraries like yt-dlp which will increase the app's size significantly.

That's why I thought of using redditsave.

**Comment by test2a** [comment]
@suhasdissa yeah sure. My point is, why aren't you downloading from redditsave like you are downloading a photo? Transparentyl?

**Comment by hackbot1337** [comment]
+1 for this. this is a great app but its crippled when i browse gifrecipes and oddlysatisfying.

**Comment by SuhasDissa** [comment]
This app currently only support showing user uploaded images. This means it doesn't currently support external sites like imgur.
I will try my best to support external sites too.

**Comment by SuhasDissa** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Video Download](https://trello.com/c/khf9wmIt/1-video-download)
